### DISCUSSION TITLE
v7: table panel to support coloring cells with non-numeric values

### DISCUSSION THREAD

**Discussion by w-bazan** [grafana/grafana#38378] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

can table panel support coloring cells with a fixed color if they represent a non-nunemric value, please?

**Why is this needed**:

better data presentation

**Comment by torkelo** [comment]
Duplicate of https://github.com/grafana/grafana/issues/24557

**Comment by w-bazan** [comment]
> Duplicate of #24557

Hi Torkel, thanks for taking time to look into my issue. I read the referenced issue and cannot quite figure out how is it related. It says nothing about coloring. Am I missing something? Would you mind to double check or explain a bit, please?

Thanks again

**Comment by torkelo** [comment]
sorry posted this on wrong issue :)

**Comment by addtzg** [comment]
It really would be helpful, even simple scenarios like "true" - green, "false" - red are now impossible.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
